### ISSUE TITLE
Multi-Tab experiment Parameters

### ISSUE THREAD

**Issue by MichaelWrana** [Xinhao-Deng/Website-Fingerprinting-Library#12]
Hi!  Thank you so much this repository is amazing and I'm looking forward to using it for my upcoming work!

Since this repo includes ARES multi-tab, I was wondering if you had any recommendations for which parameters to use when running ARES on 3-tab data?

I was using all the default 1-tab parameters in ARES.sh and only changing num_tabs=3, loss = BCEWithLogitsLoss, and metric = P@3.  Are these reasonable choices or do you have other hyperparameter recommendations for multi-tab ARES?

Also is Holmes 3-tab compatible?  In the original paper only 1-tab experiments are performed.

**Comment by Xinhao-Deng** [comment]
You can refer to the script below to run ARES. 
```
python -u exp/train.py \
            --dataset ${dataset} \
            --model ARES \
            --device cuda:0 \
            --num_tabs ${num_tabs} \
            --train_file mtaf_train \
            --valid_file mtaf_valid \
            --feature MTAF \
            --seq_len 8000 \
            --train_epochs 50 \
            --batch_size 512 \
            --learning_rate 2e-3 \
            --optimizer AdamW \
            --loss MultiLabelSoftMarginLoss \
            --lradj StepLR \
            --eval_metrics AUC P@${num_tabs} AP@${num_tabs} \
            --save_metric AP@${num_tabs} \
            --save_name base
```
The Holmes script currently does not support running with multiple tab settings.
